client/extensions/woocommerce/woocommerce-services/views/label-settings/label-settings.js

```diff
@@ -77,7 +78,7 @@ class ShippingLabels extends Component {
 					<p className="label-settings__credit-card-description" />
 					<PaymentMethod selected={ false } isLoading={ true } />
 					<PaymentMethod selected={ false } isLoading={ true } />
-					<Button compact />
+					<Button isSecondary />
```

Comment:
Does this button need the `const buttonClasses = classNames( 'button','is-compact');` code?

Reply:
Good question. On a second look it doesn't need any classes.
The styles are set here: https://github.com/Automattic/woocommerce-services/blob/9c6b6e1c7957c66238289a34bffb7e72fededea5/client/extensions/woocommerce/woocommerce-services/views/label-settings/style.scss#L83
Removed.